vhdbuilder/packer/components.json

```diff
@@ -52,20 +52,6 @@
       "multiArchVersions": [
         "v0.0.4.2",
         "v0.1.1"
-      ],
-      "prefetchOptimizations": [
```

Comment:
is a script being used to make these updates? if so we'll need to change it to preserve `prefetchOptimizations`